polygeist/tools/cgeist/Test/Verification/conditional.cpp

```diff
@@ -0,0 +1,41 @@
+// RUN: cgeist --use-opaque-pointers %s --function=* -S | FileCheck %s
```

Comment:
Do we really need the explicit `use-opaque-pointers` flag in this test?

Reply:
Dropping, as we are not using `llvm.ptr`